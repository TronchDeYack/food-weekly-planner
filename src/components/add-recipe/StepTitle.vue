<template>
  <v-stepper-step :complete="complete" :step="step">
    {{ title }}
  </v-stepper-step>
</template>

<script>
export default {
  name: 'add-recipe-step-title',
  props: {
    title: {
      type: String,
    },
    complete: {
      type: Boolean,
    },
    step: {
      type: Number,
    },
  },
};
</script>
