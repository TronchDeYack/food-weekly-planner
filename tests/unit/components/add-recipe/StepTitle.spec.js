import StepTitle from '@/components/add-recipe/StepTitle.vue';

import { createComponent } from '../../../utils';

describe('StepTitle', () => {
  const propsData = {
    step: 2,
    complete: true,
    title: 'DUQ',
  };

  describe('render', () => {
    it('Should render.', () => {
      const wrapper = createComponent(StepTitle, propsData);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
