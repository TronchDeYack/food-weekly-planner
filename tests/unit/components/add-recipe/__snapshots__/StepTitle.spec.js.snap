// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepTitle render Should render. 1`] = `
<v-stepper-step-stub color="primary" complete="true" completeicon="$complete" editicon="$edit" erroricon="$error" rules="" step="2">
  DUQ
</v-stepper-step-stub>
`;
